pkg/kapis/devops/v1alpha3/common/options.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 KubeSphere Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package common
```

Comment:
What is the difference between this file and the following one?

pkg/kapis/devops/v1alpha1/common/options.go

Reply:
By the way, please don't forget the license header check.

Reply:
Currently, Argo API handler is using the options, it would be deleted after moving Argo CRD to another group.